product_listings_manager/tests/test_rest_api_v1.py

```diff
@@ -60,3 +61,18 @@ def test_get_product_listings(self, mock_get_product_listings, client):
         r = client.get(path)
         assert r.status_code == 200
         assert r.get_json() == mock_get_product_listings.return_value
+
+
+class TestModuleProductListings(object):
+    product_label = 'RHEL-8.0.0'
+    nsvc = 'ruby-2.5-820181217154935.9edba152'
```

Comment:
NSVC would be `ruby:2.5:820181217154935:9edba152`, the variable name is not really accurate.